use/quickstart.md

```diff
@@ -5,28 +5,171 @@ permalink: /use/quickstart/
-* [Dataflow and open source - proposal to join the Apache Incubator](http://googlecloudplatform.blogspot.com/2016/01/Dataflow-and-open-source-proposal-to-join-the-Apache-Incubator.html)
+* TOC
+{:toc}
 
-## Current code
-The following GitHub repositories contain code which will be incorporated into Apache Beam.
 
-* [Dataflow Java SDK](https://github.com/GoogleCloudPlatform/DataflowJavaSDK)
-* [Flink Dataflow runner](https://github.com/dataArtisans/flink-dataflow)
-* [Spark Dataflow runner](https://github.com/cloudera/spark-dataflow)
+## Set up your Development Environment
+ 
+1. Download and install the [Java Development Kit (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/index.html) version 1.7 or later. Verify that the [JAVA_HOME](https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/envvars001.html) environment variable is set and points to your JDK installation.
 
-These code repositories will be refactored and managed together (along with other code and new contributions) into a single repository.
+1. Download and install [Apache Maven](http://maven.apache.org/download.cgi) by following Maven's [installation guide](http://maven.apache.org/install.html) for your specific operating system.
 
-## Documentation
-* [Apache Beam incubation proposal](https://goo.gl/KJrEl7)
-* *Apache Beam technical vision*
-    * [Detailed](https://goo.gl/5qZt3d)
-    * [Summary](https://goo.gl/nk5OM0)
-* [Apache Beam technical documentation](https://goo.gl/ps8twC)
+
+## Get the WordCount Code
+
+The easiest way to get a copy of the WordCount pipeline is to use the following command to generate a simple maven project that contains Beam's WordCount examples and builds against the most recent Beam release: 
+
+```
+$ mvn archetype:generate \
+      -DarchetypeArtifactId=beam-sdks-java-maven-archetypes-examples \
+      -DarchetypeVersion=LATEST \
+      -DarchetypeGroupId=org.apache.beam \
+      -DgroupId=org.example \
+      -DartifactId=word-count-beam \
+      -Dversion="0.1" \
+      -DinteractiveMode=false \
+      -Dpackage=org.apache.beam.examples
+```
+
+This will create a directory `word-count-beam` that contains a simple pom file and series of example pipelines that count words in text files. 
+
+```
+$ cd beam-word-count/
+
+$ ls
+pom.xml	src
+
+$ ls src/main/java/org/apache/beam/examples/
+DebuggingWordCount.java	WindowedWordCount.java	common
+MinimalWordCount.java	WordCount.java
+```
+
+For a detailed introduction to the Beam concepts used in these examples, see the [WordCount Example Walkthrough](/use/wordcount-example). 
+
+
+## Run WordCount
+
+A single Beam pipeline can run on multiple Beam [runners](/learn/runners), including [Spark](/learn/runners/spark), [Flink](/learn/runners/flink), or [Dataflow](/learn/runners/dataflow). The [DirectRunner](/learn/runners/direct) is a common runner for getting started, as it runs locally on your machine and requires no specific setup.
```

Comment:
Instead, renamed these to the runner names ('SparkRunner', etc) given that they link to those pages, not the projects' apache pages